### PR TITLE
Automated platform-alteration tainted-node-kernel tcs

### PR DESCRIPTION
**Pull request by shaior**
Fixed typo in runhelper, automated 51389, 51390

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]

File: tests/platformalteration/tests/platform_alteration_tainted_node_kernel.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+var _ = Describe("platform-alteration-tainted-node-kernel", func() {
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(tsparams.PlatformAlterationNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	const rebootWaitingTime = 10 * time.Minute
```
Comment (kononovn) [inline tests/platformalteration/tests/platform_alteration_tainted_node_kernel.go:25]: Please move it to parameters.go 
```
const (
    rebootWaitingTime = 10 * time.Minute
    reboot = `chroot /host systemctl reboot
)
```

**Comment by kononovn** [comment]
Please rebase

File: tests/platformalteration/helper/helper.go [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/client"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/nodes"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 const WaitingTime = 5 * time.Minute
+
+func WaitForSpecificNodeCondition(clients *client.ClientSet, timeout, interval time.Duration, nodeName string,
```
Comment (kononovn) [inline tests/platformalteration/helper/helper.go:16]: Please add comment

Comment (shaior) [reply tests/platformalteration/helper/helper.go:17]: Done

File: tests/platformalteration/tests/platform_alteration_tainted_node_kernel.go (hunk 1 above)
Comment (shaior) [reply tests/platformalteration/tests/platform_alteration_tainted_node_kernel.go]: Done

**Pull request merged**